src/app/globals.css

```
@@ -148,50 +146,48 @@
+  background: hsl(var(--panel) / 0.85);
   backdrop-filter: blur(12px);
   -webkit-backdrop-filter: blur(12px);
 }
 
-/* Chronicle sidebar nav item */
+/* ── Chronicle nav item ────────────────────────────────────────────── */
 .nav-item {
   @apply flex items-center gap-2.5 px-3 py-1.5 rounded-lg text-sm font-medium transition-all duration-150 cursor-pointer select-none;
-
-  color: rgba(255,255,255,0.55);
+  color: hsl(var(--muted-foreground));
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)